### PR TITLE
[phpfpm] Added ability to directly connect to a TCP address:port and modify status path

### PR DESCRIPTION
**Pull request by vobruba-martin**
With this change, you can monitor non-socket PHP-FPM instances (PHP-FPM does not allow you to listen on a socket and a port at the same time) without a webserver.

### CODE REVIEW

File: modules/phpfpm/client.go [hunk 1]
```diff
@@ -143,3 +143,50 @@ func (c *socketClient) getStatus() (*status, error) {
...
+		return nil, fmt.Errorf("error on connecting to address '%s': %v", c.address, err)
+	}
+	defer client.Close()
+
+	resp, err := client.Get(c.env)
+	if err != nil {
+		return nil, fmt.Errorf("error on getting data from address '%s': %v", c.address, err)
+	}
+
+	content, err := ioutil.ReadAll(resp.Body)
```
Comment (ilyam8) [inline modules/phpfpm/client.go:181]:
```suggestion
	content, err := io.ReadAll(resp.Body)
```

**Comment by ilyam8** [comment]
Hi, @vobruba-martin. Big thanks for the contribution!

> you can monitor non-socket PHP-FPM instances

You did test the change, didn't you?

**Comment by vobruba-martin** [comment]
Hello @ilyam8 

> You did test the change, didn't you?

Yes, I did!

**Comment by ilyam8** [comment]
@vobruba-martin try avoiding force-pushes when it is not necessary, it makes it harder to review the changes (not possible to see the changes after force-push).

**Comment by ilyam8** [comment]
@vobruba-martin the failed check is not formatted code

**Comment by vobruba-martin** [comment]
@ilyam8 I'm sorry for the force-push.

Could you please advise why the check is failing now?

**Review by ilyam8 — APPROVED** [review]

**Comment by vobruba-martin** [comment]
@ilyam8 Thank you for the review!

**Pull request merged**